src/libraries/System.Runtime/tests/System.Reflection.Tests/AssemblyTests.cs

```diff
@@ -181,6 +183,32 @@ public void GetEntryAssembly()
             Assert.True(correct, $"Unexpected assembly name {assembly}");
         }
 
+        [Fact]
```

Comment:
```suggestion
        [ConditionalTheory(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
```
to fix test failures